backend/de.metas.business/src/main/java/de/metas/project/workorder/calendar/ToCalendarEntryConverter.java

```diff
@@ -26,8 +28,10 @@
 class ToCalendarEntryConverter
 {
 	private static final Logger logger = LogManager.getLogger(ToCalendarEntryConverter.class);
-	@NonNull private final IUOMDAO uomDAO = Services.get(IUOMDAO.class);
-	@NonNull private final ProjectFrontendURLsProvider frontendURLs = new ProjectFrontendURLsProvider();
+	@NonNull
+	private final IUOMDAO uomDAO = Services.get(IUOMDAO.class);
```

Comment:
pls check your intelliJ code style:
![image](https://user-images.githubusercontent.com/1244701/183858399-e77e80b9-4827-4380-a1b6-2f6a8ce030b7.png)

@metas-ts  don't we have these settings in our team coding styles?

Reply:
not sure...at any rate, my settings also say "Field annotations" => always wrap